e2e/tests/stake_app.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Stake Namespaces
+
+    Scenario: User can stake an Application
+        Given the user has the pocketd binary installed
+        And the "application" for account "app2" is not staked
+        And the account "app2" has a balance greater than "1000070" uPOKT
```

Comment:
Please add a comment  on where this number came from. It's one more than the default IIRRC.